src/Rootfs.jl

```diff
@@ -492,13 +500,13 @@ function choose_shards(p::Platform;
         platform_match(a, b) = ((typeof(a) <: typeof(b)) && (arch(a) == arch(b)) && (libc(a) == libc(b)))
         if :c in compilers
             append!(shards, [
-                CompilerShard("GCCBootstrap", GCC_build, host_platform, archive_type; target=p),
+                make_gcc_shard(GCC_build, p),
                 CompilerShard("LLVMBootstrap", LLVM_build, host_platform, archive_type),
             ])
             # If we're not building for the host platform, then add host shard for host tools
             if !platform_match(p, host_platform)
                 append!(shards, [
-                    CompilerShard("PlatformSupport", ps_build, host_platform, archive_type; target=host_platform),
```

Comment:
Is it intentional to remove the PlatformSupport shard?

Reply:
No, I got my wires crossed. Thanks for catching that. This was supposed to replace the next line.